virttest/utils_params.py

```diff
@@ -104,3 +105,94 @@ def copy_from_keys(self, keys):
             if self.get(key):
                 new_dict[key] = self.get(key)
         return new_dict
+
+    def get_boolean(self, key, enabled_by_default=False):
+        """
+        Check if a config option is set to a default affirmation or not.
+
+        :param key: config option key
+        :type key: str
+        :param bool enabled_by_default: whether to assume "yes" or "no" if param
+                                        does not exist (defaults to False/"no").
+        :return whether option is set to 'yes'
+        :rtype: bool
+        """
+        value = self.get(key, "yes" if enabled_by_default else "no")
+        if value in ("yes", "on", "true"):
+            return True
+        if value in ("no", "off", "false"):
+            return False
+        raise ValueError("Cannot get boolean parameter value for %s: %s", key, value)
+
+    def get_numeric(self, key, default=0):
+        """
+        Get numeric value converting to integer if necessary.
+
+        :param key: config option key
+        :type key: str
+        :param int default: default numeric value
+        :return integer converted numeric value
+        :rtype: int
+        """
+        return int(self.get(key, default))
```

Comment:
Better to let user be able to specify the type instead of always using int.

Reply:
Good point, resolved in the next push.